src/sass/style.scss

```diff
@@ -32,7 +32,13 @@
 /********************/
 /* Components       */
 /********************/
+.grav-c-callout {
+  background: url('/images/callout.jpg') center center;
+  background-size: cover;
 
+  // Adds flow so that the container is wrapped properly
+  padding: 1px 0;
```

Comment:
Hmm.. leave this for now, but this suggests there's an issue upstream with the Gravity code.

Seeing the component on a page I now realise that the horizontal padding/margin isn't quite right either because the text doesn't line up with the rest of the page:
<img width="1204" alt="image" src="https://user-images.githubusercontent.com/16718916/59211239-4e3e0f00-8ba7-11e9-84c3-5cfe0ebcbdee.png">

I suggest we leave this code as is for now, amend the component's CSS in Gravity and, once we've published that as a new alpha release, we can remove this work-around.

Reply:
It looked funny to me too, I'll get this merged in, then fix it within the package.